lib/services/local/headlessauthn_watcher.go

```diff
@@ -219,33 +223,34 @@ func (h *HeadlessAuthenticationWatcher) notify(headlessAuthns ...*types.Headless
 	}
 }
 
+type WaitConditionFunc func(*types.HeadlessAuthentication) (bool, error)
```

Comment:
What does the bool mean here? Could we add a name to the return params and/or a godoc, so it's clearer?